chickadee/processes/wps_rerank.py

```diff
@@ -1,6 +1,6 @@
-import os
-import re
-from pywps import Process, LiteralInput, ComplexInput, FORMATS
+from rpy2 import robjects
+from pywps.inout.formats import Format
+from pywps import Process, ComplexInput, LiteralInput, FORMATS
```

Comment:
You do not have to import `Format` from  `pywps.inout.formats`. It is accessible from `pywps` right away.